packages/backend/discovery/starknet/ethereum/diffHistory.md

```diff
@@ -1,3 +1,121 @@
+# Diff at Mon, 18 Dec 2023 14:44:24 GMT:
+
+- author: maciekzygmunt (<maciekzygmunt@interia.pl>)
+- comparing to: master@4b160bc70449af36363ff58bf34ad3610acc00ff
+
+## Description
+
+Provide description of changes. This section will be preserved.
```

Comment:
provide a small description here of what you done

Reply:
[9110b31](https://github.com/l2beat/l2beat/pull/2392/commits/9110b31c04627e1ee6a3fd4b11a0bfc29e0eedb1)